ads/google/doubleclick.js

```diff
@@ -79,6 +79,14 @@ function doubleClickWithGpt(global, data, gladeExperiment) {
     parseInt(data.overrideHeight || data.height, 10),
   ]];
 
+  // Center the ad in the container.
+  const container = global.document.querySelector('#c');
```

Comment:
I assume that when you get here, this selector is guaranteed to return a non-null element?